### ISSUE TITLE
v2024.3.3b1 (Windows): Error 'Invalid nonce prefix(20)]' for FiroPoW@ZergPool

### ISSUE THREAD

**Issue by UselessGuru** [TrailingStop/TT-Miner-release#52]
`Bin\TTMiner-v2024.3.3b1\TT-Miner.exe -a FiroPow -o ssl://firopow.eu.mine.zergpool.com:13001 -u ###################################.############# -p c=BTC,mc=KIIRO,ID=############ -report-average 5 -report-interval 5 -daginfo -b 127.0.0.1:4002 -d 4
` 
or
`Bin\TTMiner-v2024.3.3b1\TT-Miner.exe -c KIIRO -o ssl://firopow.eu.mine.zergpool.com:13001 -u ###################################.############# -p c=BTC,mc=KIIRO,ID=############ -report-average 5 -report-interval 5 -daginfo -b 127.0.0.1:4002 -d 4
`
or
`Bin\TTMiner-v2024.3.3b1\TT-Miner.exe -a FiroPow -o stratum+ssl://firopow.eu.mine.zergpool.com:13001 -u ###################################.############# -p c=BTC,mc=KIIRO,ID=############ -report-average 5 -report-interval 5 -daginfo -b 127.0.0.1:4002 -d 4
` 
or
`Bin\TTMiner-v2024.3.3b1\TT-Miner.exe -c KIIRO -o stratum+ssl://firopow.eu.mine.zergpool.com:13001 -u ###################################.############# -p c=BTC,mc=KIIRO,ID=############ -report-average 5 -report-interval 5 -daginfo -b 127.0.0.1:4002 -d 4
`
```
07:18:47.781 Pool[KIIRO-0]: **** Summary for: KiiroCoin / FiroPoW
07:18:47.781 GPU[04-KIIRO]: *0 H/s [A0:R0:S0 0.0%]  CClk:0 Hz  MClk:0 Hz  50°C  0%  5.174 GB   -
07:18:47.781 All[KIIRO]:    *0 H/s [A0:R0:S0 0.0%]  uptime: 0 00:00:15  mining-time: 0 00:00:14  last: 00:00
07:18:47.781 Coin[KIIRO]:   *Active DAG epoch: 161, size = 2.758 GB, cache size: 36.12 MB, changes = 1
07:18:50.112 GPU[04-KIIRO]: Found solution #1, diff = 71.13 MH
07:18:50.157 GPU[04-KIIRO]: Rejected #1, RT in 45 ms. [Invalid nonce prefix(20)][firopow.eu.mine.zergpool.com:13001]
07:18:50.595 GPU[04-KIIRO]: Found solution #2, diff = 50.47 MH
07:18:50.641 GPU[04-KIIRO]: Rejected #2, RT in 45 ms. [Invalid nonce prefix(20)][firopow.eu.mine.zergpool.com:13001]
07:18:52.783 Pool[KIIRO-0]: **** Summary for: KiiroCoin / FiroPoW
07:18:52.783 GPU[04-KIIRO]: *0 H/s [A0:R2:S0 100.0%]  CClk:0 Hz  MClk:0 Hz  52°C  0%  5.174 GB   last: 00:02
07:18:52.783 All[KIIRO]:    *0 H/s [A0:R2:S0 100.0%]  uptime: 0 00:00:20  mining-time: 0 00:00:19  last: 00:02
07:18:52.783 Coin[KIIRO]:   *Active DAG epoch: 161, size = 2.758 GB, cache size: 36.12 MB, changes = 1
07:18:57.787 Pool[KIIRO-0]: **** Summary for: KiiroCoin / FiroPoW
07:18:57.787 GPU[04-KIIRO]: *7.196 MH/s [A0:R2:S0 100.0%]  CClk:0 Hz  MClk:0 Hz  54°C  0%  5.174 GB   last: 00:07
07:18:57.787 All[KIIRO]:    *7.196 MH/s [A0:R2:S0 100.0%]  uptime: 0 00:00:25  mining-time: 0 00:00:24  last: 00:07
07:18:57.787 Coin[KIIRO]:   *Active DAG epoch: 161, size = 2.758 GB, cache size: 36.12 MB, changes = 1
07:18:59.439 GPU[04-KIIRO]: Found solution #3, diff = 43.13 MH
07:18:59.484 GPU[04-KIIRO]: Rejected #3, RT in 45 ms. [Invalid nonce prefix(20)][firopow.eu.mine.zergpool.com:13001]
07:19:02.244 Pool[KIIRO-0]: *Received new job#: 141c8
07:19:02.790 Pool[KIIRO-0]: **** Summary for: KiiroCoin / FiroPoW
07:19:02.790 GPU[04-KIIRO]: *7.196 MH/s [A0:R3:S0 100.0%]  CClk:0 Hz  MClk:0 Hz  55°C  0%  5.174 GB   last: 00:03
07:19:02.790 All[KIIRO]:    *7.196 MH/s [A0:R3:S0 100.0%]  uptime: 0 00:00:30  mining-time: 0 00:00:29  last: 00:03
07:19:02.790 Coin[KIIRO]:   *Active DAG epoch: 161, size = 2.758 GB, cache size: 36.12 MB, changes = 1
07:19:07.790 Pool[KIIRO-0]: **** Summary for: KiiroCoin / FiroPoW
07:19:07.790 GPU[04-KIIRO]: *7.196 MH/s [A0:R3:S0 100.0%]  CClk:0 Hz  MClk:0 Hz  57°C  0%  5.174 GB   last: 00:08
07:19:07.790 All[KIIRO]:    *7.196 MH/s [A0:R3:S0 100.0%]  uptime: 0 00:00:35  mining-time: 0 00:00:34  last: 00:08
07:19:07.790 Coin[KIIRO]:   *Active DAG epoch: 161, size = 2.758 GB, cache size: 36.12 MB, changes = 1
07:19:08.595 GPU[04-KIIRO]: Found solution #4, diff = 1.828 GH
07:19:08.640 GPU[04-KIIRO]: Rejected #4, RT in 45 ms. [Invalid nonce prefix(20)][firopow.eu.mine.zergpool.com:13001]
07:19:09.427 GPU[04-KIIRO]: Found solution #5, diff = 59.54 MH
07:19:09.471 GPU[04-KIIRO]: Rejected #5, RT in 44 ms. [Invalid nonce prefix(20)][firopow.eu.mine.zergpool.com:13001]
```

Same with non-SSL.
Other miners - I tested Trex - are working fine.

**Comment by TrailingStop** [comment]
yes- thanks. this is a bug.

**Comment by UselessGuru** [comment]
BTW: I get the same **[Invalid nonce prefix(20)]** error when mining SCCpow.

**Comment by TrailingStop** [comment]
Yes. same protocol. do you use windows or linux. I will make a quick testversion to verify the fix if I'm back.

**Comment by TrailingStop** [comment]
Hi- please check out the latest beta (2024.3.3-beta4). Bug is fixed here - please verify and let me know.
Thanks

**Comment by UselessGuru** [comment]
Fixed in 2024.3.3-beta4
